src/selectors/getResultData.test.ts

```diff
@@ -0,0 +1,173 @@
+        globalConfig: {
+            timeout: 6000
+        },
+        endpointConfig: {
+            getData: {
+                url: 'https://myapi.org/myData',
+                method: 'GET'
+            }
+        },
+        requests: {
+            [getRequestKey('getData')]: {
+                networkStatus: 'success',
+                lastCall: Date.now(),
+                duration: 6000,
+                endpointKey: 'getData',
+                result: 1
+            }
+        },
+        entities: {
+            users: {
+                '123abc': 'oberon'
+            },
+            articles: {
+                1: {
+                    id: 1,
+                    comments: [1, 2],
+                },
+            },
+            comments: {
+                1: {
+                    id: 1,
+                    content: 'Comment 1',
+                }, 2: {
+                    id: 2,
+                    content: 'Comment 2',
+                }
+            }
+        }
+    };
+
+    const result = getResultData(apiDataState2, 'getData');
+
+    expect(result).toEqual(1);
+});
+
+test('should return denormalized result data', () => {
+
+    const commentsSchema = new schema.Entity('comments');
+    const articlesSchema = new schema.Entity('articles', {
+        comments: [commentsSchema]
+    });
+
+    const apiDataState3: ApiDataState = {
+        globalConfig: {
+            timeout: 6000
+        },
+        endpointConfig: {
+            getData: {
+                url: 'https://myapi.org/myData',
+                method: 'GET',
+                responseSchema: articlesSchema
+            }
+        },
+        requests: {
+            [getRequestKey('getData')]: {
+                networkStatus: 'success',
+                lastCall: Date.now(),
+                duration: 6000,
+                endpointKey: 'getData',
+                result: 1
+            }, [getRequestKey('getData', { id: 'one' })]: {
+                networkStatus: 'success',
+                lastCall: Date.now(),
+                duration: 6000,
+                endpointKey: 'getData'
+            }
+        },
+        entities: {
+            users: {
+                '123abc': 'oberon'
+            },
+            articles: {
+                1: {
+                    id: 1,
+                    comments: [1, 2],
+                },
+            },
+            comments: {
+                1: {
+                    id: 1,
+                    content: 'Comment 1',
+                }, 2: {
+                    id: 2,
+                    content: 'Comment 2',
+                }
+            }
+        }
+    };
+
+    const result = getResultData(apiDataState3, 'getData');
+
+    expect(result).toEqual({
+        id: 1,
+        comments: [
+            {
+                id: 1,
+                content: 'Comment 1',
+            },
+            {
+                id: 2,
+                content: 'Comment 2',
+            }
+        ]
+    });
+});
```

Comment:
pls add a test for the use of the params argument. Also for not not found results (undefined, no error)